src/kube.rs

```diff
@@ -327,6 +327,24 @@ pub struct SecretList {
     pub items: Vec<Value>,
 }
 
+// Jobs
+#[derive(Debug, Deserialize)]
+pub struct JobSpec {
+    pub parallelism: u32,
+    pub completions: u32,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct Job {
+    pub metadata: Metadata,
+    pub spec: JobSpec,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct JobList {
+    pub items: Vec<Job>,
```

Comment:
As I've worked on Click I've realized k8s is super inconsistent about which fields it returns via the api.  As such I've moved away from fully specifying objects like this since missing fields will cause Click to just exit with an error from the serde code in that case.  Rather I just specify a `Vec<Value>` and pull out the info I want to display from the `Value`, with a default.  You can see an example in how I handle ConfigMaps or Secrets.  This is a bit less efficient but way more resilient.

That being said, I'd like to get this merged, so if you don't want to re-write this bit we can leave it for now and then move it to using `Value` as I clean up some of the other ones.